outputs.tf

```diff
@@ -17,3 +17,13 @@ output "dl_queue_url" {
   description = "Dead letter queue url."
   value       = one(aws_sqs_queue.dlqueue[*].url)
 }
+
+output "lambda_function_arn" {
+  description = "Lambda function arn."
+  value       = aws_lambda_function.this[*].lambda_function_arn
```

Comment:
Sorry to be pedantic, but the module creates a single , non-optional, lambda function.  The use of the splat expression is not correct in this scenario

Reply:
Removing the splat results in a validation error:

```
{
      "severity": "error",
      "summary": "Unsupported attribute",
      "detail": "This object has no argument, nested block, or exported attribute named \"lambda_function_arn\".",
      "range": {
        "filename": "outputs.tf",
        "start": {
          "line": 23,
          "column": 41,
          "byte": 548
        },
        "end": {
          "line": 23,
          "column": 61,
          "byte": 568
        }
      },
      "snippet": {
        "context": "output \"lambda_function_arn\"",
        "code": "  value       = aws_lambda_function.this.lambda_function_arn",
        "start_line": 23,
        "highlight_start_offset": 40,
        "highlight_end_offset": 60,
        "values": []
      }
    },
```

Reply:
@peasee I think the attribute reference for arn is just .arn.

The lambda function name isn't an attribute you can reference either I think. So probably just want to include if exist.

Reply:
Also for name the user already knows about it since we pass `var.name`. Not sure if it's worth including an output. If so it will just be the value of `var.name`.

Reply:
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function#attributes-reference